op-monitorism/global_events/rules/rules_mainnet_L1/Initialized-Upgraded.yaml

```diff
@@ -0,0 +1,17 @@
+version: 1.0
+name: Initialized Upgraded Proxy L1
+priority: P2
+addresses:
+  - 0x9ba6e03d8b90de867373db8cf1a58d2f7f006b3a
```

Comment:
This is a safe address, so it can't emit these events. Doesn't hurt to include, but I think it's less confusing to exclude it.

Reply:
Thanks! I have kept in case the multisig initiate a call (the goal is to monitore more widely the first days, week). As we need to make a clean rules reviewed anyway (that should be not long but for example adding comments + merging the same addresses in the same rules cf: security council) until this moment I would like to kept it.